evaluator/functions.go

```diff
@@ -411,6 +416,28 @@ func pwdFn(args ...object.Object) object.Object {
 	return &object.String{Token: tok, Value: dir}
 }
 
+// cd(path)
+func cdFn(args ...object.Object) object.Object {
+	// Default cd $HOME
+	path := os.Getenv("HOME")
+	if len(args) == 1 {
+		// arg: path
+		pathStr := args[0].(*object.String)
+		rawPath := pathStr.Value
+		// expand ~/ path prefix
+		if strings.HasPrefix(rawPath, "~/") {
+			path = strings.Replace(rawPath, "~/", path+"/", 1)
```

Comment:
I think this might not work under windows :)

Reply:
Good call. I'll make it portable.